index.html

```diff
@@ -1116,28 +1116,12 @@ <h2><dfn>Populating and maintaining the coalesced and predicted event lists</dfn
                 <li>Initialize the rest of the attributes the same way as <a>PointerEvent</a>.</li>
             </ol>
 
-            <p>A <a>PointerEvent</a> has an associated <dfn>coalesced events targets dirty</dfn> and an associated
-            <dfn>predicted events targets dirty</dfn> flag. When an event is created they must be initialized to false.</p>
-
-            <p>When the <a>PointerEvent</a>'s target is changed, set <a><code>coalesced events targets dirty</code></a>
-            and <a><code>predicted events targets dirty</code></a> to true.</p>
-
-            <p>When the <code><a data-lt="PointerEvent.getCoalescedEvents">getCoalescedEvents</a></code> method is called:</p>
+            <p>When a trusted <a>PointerEvent</a>'s target is changed:</p>
             <ol>
-                <li>If the <a>coalesced events targets dirty</a> is true:
-                for each event in the <a>coalesced event list</a>, set the event's {{Event/target}}
+                <li>for each event in the <a>coalesced event list</a>, set the event's {{Event/target}}
                 to this <code>PointerEvent</code>'s target.</li>
```

Comment:
This was there before but never came to me: the "this" here seems unclear to me.  The first point in the above para (source line 1110) doesn't use "this", but also unclear to me :( .
- Can we make both paras consistent?
- Can we perhaps use something like "container event"?

Reply:
keep in mind that this PR will then be followed by https://github.com/w3c/pointerevents/pull/377 which completely rejigs this stuff....